code/modules/mob/living/carbon/carbon.dm

```diff
@@ -995,9 +993,8 @@
 
 
 /mob/living/carbon/proc/create_internal_organs()
-	for(var/X in internal_organs)
-		var/obj/item/organ/I = X
-		I.Insert(src)
+	for(var/obj/item/organ/internal/internal_organ in internal_organs)
```

Comment:
was there a reason this was pathed to `/obj/item/organ` before instead of `/obj/item/organ/internal`?

Reply:
because internal didn't exist when this was made. internal_organ only stores internal.